drivers/SmartThings/zigbee-presence-sensor/src/init.lua

```diff
@@ -159,7 +159,7 @@ local function all_zigbee_message_handler(self, message_channel)
   if zb_rx ~= nil then
     device.log.info(string.format("received Zigbee message: %s", zb_rx:pretty_print()))
     device:attribute_monitor(zb_rx)
-    poke(device)
+    if device:get_model() ~= "lumi.motion.ac01" then poke(device) end -- excluding Aqara PresenceSensor FP1
```

Comment:
I'd recommend over-writing this handler in your sub-driver with an empty function instead, so that all relevant code can be kept in the sub-driver

Reply:
I tried overwriting the zigbee_message_handler handler as follows. 
However, the handler defined in the sub-driver does not work.
Could you give me a guide or example code so I can understand?

```
local aqara_fp1_handler = {
  NAME = "Aqara Presence Senser FP1 Handler",
  zigbee_handlers = {
    attr = {
      [PRIVATE_CLUSTER_ID] = {
        [PRESENCE_ATTRIBUTE_ID] = presence_monitor_handler,
        [MOVEMENT_ATTRIBUTE_ID] = move_monitor_handler
      }
    }
  },
  lifecycle_handlers = {
    init = device_init,
    added = device_added,
    infoChanged = device_info_changed
  },
  -- Custom handler for every Zigbee message
  zigbee_message_handler = all_zigbee_message_handler,
  can_handle = is_aqara_products
}
```

Reply:
please check it out.

Reply:
Hm. Maybe it only applies to top level drivers.